### PR TITLE
Update dependencies and move eslint in devDependencies.

### PR DESCRIPTION
**Pull request by XhmikosR**
Requires #133, I'll rebase after that is merged.

### CODE REVIEW

**Comment by XhmikosR** [comment]
@jonrohan this should be merged before #138

**Comment by XhmikosR** [comment]
Note that I cannot debug the deployments myself.

**Comment by jonrohan** [comment]
Looks like this is failing because it needs Node 12

```
Installing build runtime...
Build runtime installed: 6878.559ms
[36mINFO[0m[0000] Resolved base name node:9.11.2-alpine to node:9.11.2-alpine 
[36mINFO[0m[0000] Resolved base name node:9.11.2-alpine to node:9.11.2-alpine 
[36mINFO[0m[0000] Downloading base image node:9.11.2-alpine    
[36mINFO[0m[0001] Error while retrieving image from cache: getting file info: stat /cache/sha256:c812a92b98800c243bd4bafbc5a528b4fdf4bcd57a1d8de4192deded085ba6da: no such file or directory 
[36mINFO[0m[0001] Downloading base image node:9.11.2-alpine    
[36mINFO[0m[0001] Built cross stage deps: map[]                
[36mINFO[0m[0001] Downloading base image node:9.11.2-alpine    
[36mINFO[0m[0002] Error while retrieving image from cache: getting file info: stat /cache/sha256:c812a92b98800c243bd4bafbc5a528b4fdf4bcd57a1d8de4192deded085ba6da: no such file or directory 
[36mINFO[0m[0002] Downloading base image node:9.11.2-alpine    
[36mINFO[0m[0003] Executing 0 build triggers                   
[36mINFO[0m[0003] Unpacking rootfs as cmd RUN cp -r ~ /home/nowuser requires it. 
[36mINFO[0m[0005] Taking snapshot of full filesystem...        
[36mINFO[0m[0010] WORKDIR /home/nowuser                        
[36mINFO[0m[0010] cmd: workdir                                 
[36mINFO[0m[0010] Changed working directory to /home/nowuser   
[36mINFO[0m[0010] Creating directory /home/nowuser             
[36mINFO[0m[0010] RUN cp -r ~ /home/nowuser                    
[36mINFO[0m[0010] cmd: /bin/sh                                 
[36mINFO[0m[0010] args: [-c cp -r ~ /home/nowuser]             
[36mINFO[0m[0010] ENV HOME=/home/nowuser                       
[36mINFO[0m[0010] RUN apk add --update --no-cache --virtual .gyp gcc g++ make python git 
[36mINFO[0m[0010] cmd: /bin/sh                                 
[36mINFO[0m[0010] args: [-c apk add --update --no-cache --virtual .gyp gcc g++ make python git] 
fetch http://dl-cdn.alpinelinux.org/alpine/v3.6/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.6/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.6/community/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.6/community/x86_64/APKINDEX.tar.gz
(1/32) Upgrading musl (1.1.16-r14 -> 1.1.16-r15)
(2/32) Installing binutils-libs (2.30-r1)
(3/32) Installing binutils (2.30-r1)
(4/32) Installing gmp (6.1.2-r0)
(5/32) Installing isl (0.17.1-r0)
(6/32) Installing libgomp (6.3.0-r4)
(7/32) Installing libatomic (6.3.0-r4)
(8/32) Installing pkgconf (1.3.7-r0)
(9/32) Installing mpfr3 (3.1.5-r0)
(10/32) Installing mpc1 (1.0.3-r0)
(11/32) Installing gcc (6.3.0-r4)
(12/32) Installing musl-dev (1.1.16-r15)
(13/32) Installing libc-dev (0.7.1-r0)
(14/32) Installing g++ (6.3.0-r4)
(15/32) Installing make (4.2.1-r0)
(16/32) Installing libbz2 (1.0.6-r5)
(17/32) Installing expat (2.2.0-r1)
(18/32) Installing libffi (3.2.1-r3)
(19/32) Installing gdbm (1.12-r0)
(20/32) Installing ncurses-terminfo-base (6.0_p20171125-r1)
(21/32) Installing ncurses-terminfo (6.0_p20171125-r1)
(22/32) Installing ncurses-libs (6.0_p20171125-r1)
(23/32) Installing readline (6.3.008-r5)
(24/32) Installing sqlite-libs (3.25.3-r0)
(25/32) Installing python2 (2.7.15-r0)
(26/32) Installing ca-certificates (20161130-r3)
(27/32) Installing libssh2 (1.8.2-r0)
(28/32) Installing libcurl (7.61.1-r2)
(29/32) Installing pcre (8.41-r0)
(30/32) Installing git (2.13.7-r2)
(31/32) Installing .gyp (0)
(32/32) Upgrading musl-utils (1.1.16-r14 -> 1.1.16-r15)
Executing busybox-1.26.2-r11.trigger
Executing ca-certificates-20161130-r3.trigger
OK: 229 MiB in 45 packages
[36mINFO[0m[0013] RUN apk add vips-dev fftw-dev build-base --no-cache 	--repository http://dl-3.alpinelinux.org/alpine/edge/community 	--repository https://dl-3.alpinelinux.org/alpine/edge/testing 	--repository https://dl-3.alpinelinux.org/alpine/edge/main 
[36mINFO[0m[0013] cmd: /bin/sh                                 
[36mINFO[0m[0013] args: [-c apk add vips-dev fftw-dev build-base --no-cache 	--repository http://dl-3.alpinelinux.org/alpine/edge/community 	--repository https://dl-3.alpinelinux.org/alpine/edge/testing 	--repository https://dl-3.alpinelinux.org/alpine/edge/main] 
fetch https://dl-3.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://dl-3.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz
fetch http://dl-3.alpinelinux.org/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.6/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.6/community/x86_64/APKINDEX.tar.gz
WARNING: This apk-tools is OLD! Some packages might not function properly.
(1/62) Upgrading zlib (1.2.11-r0 -> 1.2.11-r3)
(2/62) Upgrading libbz2 (1.0.6-r5 -> 1.0.8-r1)
(3/62) Upgrading libffi (3.2.1-r3 -> 3.2.1-r6)
(4/62) Upgrading pcre (8.41-r0 -> 8.43-r0)
(5/62) Installing libmagic (5.37-r0)
(6/62) Installing file (5.37-r0)
(7/62) Installing fortify-headers (1.1-r0)
(8/62) Installing build-base (0.5-r1)
(9/62) Installing fftw-double-libs (3.3.8-r0)
(10/62) Installing fftw-long-double-libs (3.3.8-r0)
(11/62) Installing fftw-single-libs (3.3.8-r0)
(12/62) Installing fftw-dev (3.3.8-r0)
(13/62) Installing xz-libs (5.2.4-r0)
(14/62) Installing python3 (3.6.8-r0)
(15/62) Installing xz (5.2.4-r0)
(16/62) Installing gettext-asprintf (0.20.1-r1)
(17/62) Installing libintl (0.20.1-r1)
(18/62) Installing libunistring (0.9.10-r0)
(19/62) Installing gettext-libs (0.20.1-r1)
(20/62) Installing libxml2 (2.9.9-r2)
(21/62) Installing gettext (0.20.1-r1)
(22/62) Installing gettext-dev (0.20.1-r1)
(23/62) Installing bzip2-dev (1.0.8-r1)
(24/62) Installing libxml2-utils (2.9.9-r2)
(25/62) Installing libgpg-error (1.36-r2)
(26/62) Installing libgcrypt (1.8.5-r0)
(27/62) Installing libxslt (1.1.33-r1)
(28/62) Installing docbook-xml (4.5-r6)
Executing docbook-xml-4.5-r6.post-install
(29/62) Installing docbook-xsl (1.79.1-r1)
Executing docbook-xsl-1.79.1-r1.post-install
(30/62) Installing libblkid (2.34-r0)
(31/62) Installing libmount (2.34-r0)
(32/62) Installing glib (2.62.1-r0)
(33/62) Installing linux-headers (4.19.36-r0)
(34/62) Installing libffi-dev (3.2.1-r6)
(35/62) Installing libpcre16 (8.43-r0)
(36/62) Installing libpcre32 (8.43-r0)
(37/62) Installing libpcrecpp (8.43-r0)
(38/62) Installing pcre-dev (8.43-r0)
(39/62) Installing libuuid (2.34-r0)
(40/62) Installing libfdisk (2.34-r0)
(41/62) Installing libsmartcols (2.34-r0)
(42/62) Installing util-linux-dev (2.34-r0)
(43/62) Installing zlib-dev (1.2.11-r3)
(44/62) Installing glib-dev (2.62.1-r0)
(45/62) Installing libjpeg-turbo (2.0.3-r0)
(46/62) Installing libjpeg-turbo-dev (2.0.3-r0)
(47/62) Installing tiff (4.0.10-r1)
(48/62) Installing libtiffxx (4.0.10-r1)
(49/62) Installing tiff-dev (4.0.10-r1)
(50/62) Installing lcms2 (2.9-r1)
(51/62) Installing lcms2-dev (2.9-r1)
(52/62) Installing libexif (0.6.21-r3)
(53/62) Installing libexif-dev (0.6.21-r3)
(54/62) Installing libpng (1.6.37-r1)
(55/62) Installing libpng-dev (1.6.37-r1)
(56/62) Installing libwebp (1.0.3-r0)
(57/62) Installing libwebp-dev (1.0.3-r0)
(58/62) Installing orc (0.4.30-r0)
(59/62) Installing orc-dev (0.4.30-r0)
(60/62) Installing giflib (5.2.1-r0)
(61/62) Installing vips (8.8.2-r0)
(62/62) Installing vips-dev (8.8.2-r0)
Executing busybox-1.26.2-r11.trigger
Executing glib-2.62.1-r0.trigger
No schema files found: doing nothing.
OK: 365 MiB in 103 packages
[36mINFO[0m[0052] ARG NOW_GITHUB_DEPLOYMENT                    
[36mINFO[0m[0052] ARG NOW_GITHUB_ORG                           
[36mINFO[0m[0052] ARG NOW_GITHUB_REPO                          
[36mINFO[0m[0052] ARG NOW_GITHUB_COMMIT_ORG                    
[36mINFO[0m[0052] ARG NOW_GITHUB_COMMIT_REPO                   
[36mINFO[0m[0052] ARG NOW_GITHUB_COMMIT_REF                    
[36mINFO[0m[0052] ARG NOW_GITHUB_COMMIT_SHA                    
[36mINFO[0m[0052] ARG NOW_GITHUB_COMMIT_AUTHOR_LOGIN           
[36mINFO[0m[0052] ARG NOW_GITHUB_COMMIT_AUTHOR_NAME            
[36mINFO[0m[0052] Using files from context: [/zeit/711e1abb]   
[36mINFO[0m[0052] COPY . .                                     
[36mINFO[0m[0052] RUN yarn                                     
[36mINFO[0m[0052] cmd: /bin/sh                                 
[36mINFO[0m[0052] args: [-c yarn]                              
yarn install v1.5.1
info No lockfile found.
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error eslint@6.5.1: The engine "node" is incompatible with this module. Expected version "^8.10.0 || ^10.13.0 || >=11.10.1".
error An unexpected error occurred: "Found incompatible module".
info If you think this is a bug, please open a bug report with the information provided in "/home/nowuser/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
error building image: error building stage: waiting for process to exit: exit status 1
Error: Exited with 1
    at ChildProcess.child.on (/zeit/4cc9d509/.build-utils/.builder/node_modules/@zeit-v1-read-only/now-builder-v1/dist/utils/await-child.js:23:31)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
worker exited with code 20 and signal null
done
```

File: package.json [hunk 1]
```diff
@@ -8,19 +8,20 @@
     "build": ""
   },
   "engines": {
-    "node": "^9.3.0",
-    "npm": "^6.0.0"
+    "node": "12.x",
```
Comment (jonrohan) [inline package.json]:
```suggestion
    "node": "12.x"
```

**Comment by XhmikosR** [comment]
Ok so it needs a specific scheme. I'll change it tomorrow

**Comment by XhmikosR** [comment]
@jonrohan I reverted the eslint update for now so that we move forward. Eventually you should switch to 10 or preferably 12.

**Pull request merged**